server/src/service/machinesSchema.ts

```diff
@@ -0,0 +1,75 @@
+import { gql } from "apollo-server-express";
+
+export const MachinesTypeDefs = gql`
+  type Machine {
+    id: ID!
+    machineFamily: Int!
+    name: String!
+    room: String!
+    addedAt: Date
+    inUse: Boolean!
+  }
+
+  type MachineFamily {
+    id: ID!
+    name: String!
+    description: String
+    trainingModule: Int
+  }
+
+  type Reservation {
```

Comment:
I thought we were making it to reservations had an associated expert as well?

Reply:
like a labbie?